components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -1963,6 +1973,18 @@ private void parseOpenIDConnectConfig(OMElement oauthConfigElem) {
                                 .getText().trim();
             }
 
+            if (openIDConnectConfigElem.getFirstChildWithName(getQNameWithIdentityNS(ConfigElements.ENCRYPTION_ALGORITHM)) != null) {
```

Comment:
Length seems to be exceeding 120 charactors

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/960